### PR TITLE
Flow: complete types first migration

### PR DESCRIPTION
**Pull request by kassens**
This complete the "types first" migration and enables the config everywhere.

### CODE REVIEW

**Review by gaearon — APPROVED** [review]

File: packages/react-devtools-shared/src/node_modules/react-window/src/FixedSizeList.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 // @flow
 
+import type {ComponentType} from "react";
+
 import createListComponent from './createListComponent';
 
 import type { Props, ScrollToAlign } from './createListComponent';
 
-const FixedSizeList = createListComponent({
```
Comment (gaearon) [inline packages/react-devtools-shared/src/node_modules/react-window/src/FixedSizeList.js:7]: This folder is vendored. We shouldn't be changing it?

Comment (gaearon) [reply packages/react-devtools-shared/src/node_modules/react-window/src/FixedSizeList.js:7]: I guess we already changed it once before. Not sure why it's vendored.

Comment (kassens) [reply packages/react-devtools-shared/src/node_modules/react-window/src/FixedSizeList.js:7]: Yeah, not sure what's going on here. Maybe it should just be installed as a regular npm dependency? I suspect @bvaughn forked something in there or it was a pre-release at some point.

Comment (bvaughn) [reply packages/react-devtools-shared/src/node_modules/react-window/src/FixedSizeList.js:7]: No idea. Looking back through the history, I found 6ff733873c662086f919baced2bb80fcffc1b343 which is useless (yay me). Sorry 😄

We'd have to diff source between the vendored and non-vendored one to see what I changed.

**Pull request merged**